data/mobility.csv

```diff
@@ -77440,6 +77440,8 @@ CA,Canada,,,,,,ChIJ2WrMN9MDDUsRpY9Doiq3aJk,2021-04-18,-30,-6,73,-53,-16,8
 CA,Canada,,,,,,ChIJ2WrMN9MDDUsRpY9Doiq3aJk,2021-04-19,-31,-8,45,-59,-38,14
 CA,Canada,,,,,,ChIJ2WrMN9MDDUsRpY9Doiq3aJk,2021-04-20,-28,1,28,-57,-37,14
 CA,Canada,,,,,,ChIJ2WrMN9MDDUsRpY9Doiq3aJk,2021-04-21,-31,-5,15,-58,-38,15
+<<<<<<< HEAD
```

Comment:
I think these lines are the result of an unresolved conflict. Do you mind removing them? I'm not sure what the easiest way to do this is, but I think just copy and pasting this csv file from the main branch should be good enough.